src/notificationcenter.cs

```diff
@@ -103,14 +103,21 @@ interface NSExtensionContext_NCWidgetAdditions {
 	[BaseType (typeof (UIVibrancyEffect))]
 	interface UIVibrancyEffect_NCWidgetAdditions {
 		[iOS (10,0)]
+		[Deprecated (PlatformName.iOS, 13,0, message: "Use 'UIVibrancyEffect.GetWidgetEffect' instead.")]
 		[Static]
 		[Export ("widgetPrimaryVibrancyEffect")]
 		UIVibrancyEffect GetWidgetPrimaryVibrancyEffect ();
 
 		[iOS (10,0)]
+		[Deprecated (PlatformName.iOS, 13,0, message: "Use 'UIVibrancyEffect.GetWidgetEffect' instead.")]
 		[Static]
 		[Export ("widgetSecondaryVibrancyEffect")]
 		UIVibrancyEffect GetWidgetSecondaryVibrancyEffect ();
+
+		[iOS (13,0)]
+		[Static]
+		[Export ("widgetEffectForVibrancyStyle:")]
+		UIVibrancyEffect GetWidgetEffect (UIVibrancyEffectStyle vibrancyStyle);
```

Comment:
@mandel-macaque here

Reply:
This is a special case (that can be bound in different ways) and `xtro` is not sure
Add it (like the other already are) inside the file `tests/xtro-sharpie/iOS-NotificationCenter.ignore`